src/Presentation/Nop.Web.Framework/Infrastructure/Extensions/DbContextOptionsBuilderExtensions.cs

```diff
@@ -25,10 +27,17 @@ public static void UseSqlServerWithLazyLoading(this DbContextOptionsBuilder opti
 
             var dbContextOptionsBuilder = optionsBuilder.UseLazyLoadingProxies();
 
+            var sqlConnection = new SqlConnection(dataSettings.DataConnectionString);
```

Comment:
This is a bad idea, isn't it. I think this will use the same connection everywhere.